### ISSUE TITLE
Github repository to publish clojure koan changes to.

### ISSUE THREAD

**Issue by melissavoegeli** [clojurebridge-minneapolis/installfest#74]
I guess it would be pretty rad to introduce people to the usual git cadence of committing and pushing changes to a published repo in github.

**Comment by tmarble** [comment]
Yes! We actually forked koans https://github.com/clojurebridge-minneapolis/clojure-koans for previous ClojureBridges and upstream accepted our changes! So now our repo is rebased and we could take PR's from students/TA's immediately (and then forward them upstream).

**Comment by crkoehnen** [comment]
I love the idea, especially if you branch too.
